src/games/tristan_nogo.h

```diff
@@ -66,8 +66,9 @@ extern bool useCode;
 
 class NogoMove {
  public:
-  int inter, color;
-  int code;
+  int inter = 0;
```

Comment:
is there a reason to specify a default initialization?

Reply:
Yes, otherwise the memory is sometimes un-initialized. Which sets the nasal demons free. Thank Valgrind for the warning.

Possibly we should explicitly write a constructor that sets it to 0, idk.

Reply:
sigh, so their code had undefined behaviour and they were just lucky?

Reply:
welp